### PR TITLE
Update Node tabs

### PR DESCRIPTION
**Pull request by Mykhailenko**
- Make total nodes count mean ALL possible nodes
 - Add Needed counter to Nodes tab in RM portal
 - Arrange Nodes tab the same way Nodes State

### CODE REVIEW

File: rm-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/rm/client/RMModelImpl.java [hunk 1]
```diff
@@ -440,7 +440,7 @@ public void setNumNeeded(int pendingTasksCount) {
 
     @Override
     public int getNumNodes() {
-        return numFree + numBusy + numDown;
+        return numFree + numConfiguring + numDeploying + numLost + numBusy + numDown + numToBeRemoved;
```
Comment (mbenguig) [inline rm-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/rm/client/RMModelImpl.java]: You confirm numToBeRemoved does not include one of the other numXxx?

Comment (Mykhailenko) [reply rm-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/rm/client/RMModelImpl.java]: I confirm

File: rm-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/rm/client/StatisticsView.java [hunk 2]
```diff
@@ -104,64 +107,44 @@ protected String getCellCSSText(ListGridRecord record, int rowNum, int colNum) {
         return root;
     }
 
+    private ListGridRecord createListGridRecord(String status, String type, ImageResource icon, int count) {
```
Comment (MaelAudren) [inline rm-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/rm/client/StatisticsView.java]: Maybe it is better to have the prototype method that end with icon so both method createListGridRecord have the three first parameters.

Comment (Mykhailenko) [reply rm-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/rm/client/StatisticsView.java]: Good point thanks

**Pull request merged**